app/controllers/root_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RootController < ApplicationController
+  def index; end
+
+  def api_show
+    @messages = Greeting.first
+    render json: @messages
```

Comment:
- [ ] You are trying to fetch the first message but you are expected to fetch one **random** greeting. Kindly fix that.

  You can use this if you wish: `Greeting.all.sample`

Reply:
Thanks, @TedLivist for pointing that out. I will fix it immediately.